standalone-metastore/metastore-common/src/test/java/org/apache/hadoop/hive/common/metrics/metrics2/TestCodahaleReportersConf.java

```diff
@@ -149,4 +151,31 @@ public void testNoFallbackOnIncorrectConf() throws Exception {
 
     Assert.assertFalse(jsonReportFile.exists());
   }
+
+  /**
+   * Tests if MetricsFactory is initialised properly, if in case Metrics2Reporter is tried to add more than once.
```

Comment:
nit: 'if in case': you mean 'if' or 'in case', not both

Reply:
I mean 'if'. I will update the comment